main.py

```diff
@@ -2,6 +2,28 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+from db.data import DATA
+
 
 def main() -> QuerySet:
-    pass
+    genres, actors = DATA
+
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for actor in actors:
+        first_name, last_name = actor.split()
```

Comment:
It's better to store first name and last name in the tuples according to the checklist because the split method takes an extra time